tests/e2e/test_signing.js

```diff
@@ -18,141 +18,66 @@ const setup_conductor = require("../setup_conductor.js");
 const { Codec, KeyManager } = require('@holo-host/cryptolib');
 const { init } = require("../../src/shim.js");
 const crypto = require('crypto')
-const WH_SERVER_PORT = path.resolve(__dirname, '../tmp/shim/socket');
-const LAIR_SOCKET = path.resolve(__dirname, '../tmp/keystore/socket');
-const installedAppIds = yaml.load(fs.readFileSync('./tests/app-config.yml'));
-// NOTE: the test app servicelogger installed_app_id is hard-coded, but intended to mirror our standardized installed_app_id naming pattern for each servicelogger instance (ie:`${hostedAppHha}::servicelogger`)
-const HOSTED_APP_SERVICELOGGER_INSTALLED_APP_ID = installedAppIds[0].app_name;
+const WH_SERVER_PORT = path.resolve(__dirname, '../../script/install-bundles/shim/socket');
+const LAIR_SOCKET = path.resolve(__dirname, '../../script/install-bundles/keystore/socket');
+const installedAppIds = yaml.load(fs.readFileSync('./script/app-config.yml'));
+const INSTALLED_APP_ID = installedAppIds[2].app_name;
 
 describe("Wormhole tests", () => {
-  let shim, appWs, seed, keys, slCellId;
+  let shim, appWs, seed, keys, testCellId;
   before(async function() {
     this.timeout(100_000);
-
+    await setup_conductor.setup_conductor()
     log.info("Waiting for Lair to spin up");
-    setup_conductor.start_lair()
+    await setup_conductor.start_lair()
     await delay(5000);
-    seed = crypto.randomBytes(32);
-    keys = new KeyManager(seed);
     shim = await init(LAIR_SOCKET, WH_SERVER_PORT, async function(pubkey, message) {
+      console.log("Test shim...");
       return null;
     });
     await delay(5000);
 
     log.info("Waiting for Conductor to spin up");
-    setup_conductor.start_conductor()
+    await setup_conductor.start_conductor()
     await delay(10000);
-
     appWs = await AppWebsocket.connect('ws://localhost:42233')
-    slCellId = await setUpServicelogger(appWs)
-
+    testCellId = await getTestCellID(appWs)
   });
   after(async () => {
     await shim.stop();
     await setup_conductor.stop_conductor();
+    await setup_conductor.stop_lair();
     await resetTmp();
   });
 
-  it("test shim signing for zome call", async () => {
-    const payload = await getPayload(keys);
-    loggedActivity = await appWs.callZome({
-      cell_id: [Buffer.from(slCellId[0]), Buffer.from(slCellId[1])],
-      zome_name: 'service',
-      fn_name: 'log_activity',
-      payload,
-      cap: null,
-      provenance: Buffer.from(slCellId[1])
-    });
+  it.skip("test shim signing for zome call", async () => {
```

Comment:
Do you think we need to get this test passing before merging?

Reply:
I was just testing something. This test does pass locally. Fixing on CI soon